tests/osgi-tests/src/test/java/org/wso2/carbon/osgi/startupresolver/PendingCapabilityRegistrationOSGiTest.java

```diff
@@ -59,30 +60,26 @@
      */
     @Configuration
     public Option[] createConfiguration() {
-
-        List<Option> optionList = new ArrayList<>();
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.transport.mgt").groupId("org.wso2.carbon")
-                .versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.transport.http").groupId("org.wso2.carbon")
-                .versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.transport.jetty").groupId("org.wso2.carbon")
-                .versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.transport.custom2").groupId("org.wso2" +
-                ".carbon").versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.datasource.mgt").groupId("org.wso2.carbon")
-                .versionAsInProject());
-        optionList.add(mavenBundle().artifactId("org.wso2.carbon.sample.order.resolver").groupId("org.wso2.carbon")
-                .versionAsInProject());
-
-        optionList = OSGiTestConfigurationUtils.getConfiguration(optionList, null);
-        return optionList.toArray(new Option[optionList.size()]);
+        return new Option[] { copyDropinsBundle(
+                maven().artifactId("org.wso2.carbon.sample.transport.mgt").groupId("org.wso2.carbon")
```

Comment:
Try to have the line breaks in the end if statements, breaking lines in the middle affects the readability of the code.
Here try to break the line at "comma"